builtin/game/chatcommands.lua

```diff
@@ -530,11 +530,16 @@ core.register_chatcommand("giveme", {
 })
 
 core.register_chatcommand("spawnentity", {
-	params = "<EntityName>",
-	description = "Spawn entity at your position",
+	params = "<EntityName> [<X>,<Y>,<Z>]",
```

Comment:
two with [<>]:
https://github.com/beyondlimits/minetest/blob/master/builtin/game/chatcommands.lua#L589
https://github.com/beyondlimits/minetest/blob/master/builtin/game/chatcommands.lua#L642

two with []:
https://github.com/beyondlimits/minetest/blob/master/builtin/game/chatcommands.lua#L417
https://github.com/beyondlimits/minetest/blob/master/builtin/game/chatcommands.lua#L824

There are perhaps more.
But I think adding the <> is good. At least if we follow the system that this command uses:
https://github.com/beyondlimits/minetest/blob/master/builtin/game/chatcommands.lua#L66
Here, you can either enter "all", "privs", or some command name. Because the last one isnt meant literally, its enclosed in <>s